mmseg/datasets/a2d2.py

```diff
@@ -0,0 +1,29 @@
+import os.path as osp
+
+from .builder import DATASETS
+from .custom import CustomDataset
+
+
+@DATASETS.register_module()
+class A2D2Dataset(CustomDataset):
+    """Cityscapes dataset.
```

Comment:
A2D2 dataset

Reply:
We can add more descriptions, such as "following the Cityscapes 'trainids' format".

Reply:
Good catch! I improved the docstring and emphasized the Cityscapes 'trainids' label format.